tools/build/configure.py

```diff
@@ -23,11 +23,21 @@
 # if TYPE_CHECKING:
 #     from
 
+
 def exec_shell(command: List[str]) -> str:
     ret = subprocess.run(command, stdout=subprocess.PIPE, text=True)
     return ret.stdout
 
-def write_ninja_rules(ninja: ninja_syntax.Writer, cpp: str, extra_cppflags: str, extra_cflags: str, use_ccache: bool, shift: bool, debug: bool):
+
+def write_ninja_rules(
+    ninja: ninja_syntax.Writer,
+    cpp: str,
+    extra_cppflags: str,
+    extra_cflags: str,
+    use_ccache: bool,
+    shift: bool,
+    debug: bool,
+):
```

Comment:
Looks like you ran `black` - can we close #949 perhaps?

Reply:
That issue is to have it run consistently, so my cleanup doesn't address that